scripts/gulp/lint.js

```diff
@@ -49,12 +49,22 @@ gulp.task( 'lint:scripts', () => {
  * Lints the source LESS files for errors.
  */
 gulp.task( 'lint:styles', () => {
-  gulp.src( [ '!src/cf-grid/src-generated/*.less', 'src/**/*.less' ] )
+  // Pass all command line flags to Stylelint.
+  const options = minimist( process.argv.slice( 2 ) );
+  const willFix = options.fix || false;
+  return gulp.src( [
+      '!src/cf-*/node_modules/**/*.less',
+      '!src/cf-grid/src-generated/*.less',
+      'src/**/*.less'
+    ] )
     .pipe( gulpStylelint( {
+      failAfterError: true,
```

Comment:
Is this the bit that was being weird in our testing yesterday?

Reply:
Yeah you can tell it to fail on an error or not. I guess we want it to fail, as that would fail Travis.

Reply:
Oh good point. Only question is why isn't Travis failing now since there are errors?

Reply:
That's a good question. We'd need `gulp lint` in [here](https://github.com/cfpb/capital-framework/blob/canary/.travis.yml), right?

Reply:
Yep I think you're right. Now that I think about it, we also need to fix those errors in this PR after we've confirmed it stops Travis.

Reply:
Can be another PR. This just added the autofix flag, which Travis won't be using.

Reply:
Eh nevermind, I'll push it to this PR.

Reply:
You can see the failure in https://travis-ci.org/cfpb/capital-framework/builds/327020639?utm_source=github_status&utm_medium=notification, and it passing in https://travis-ci.org/cfpb/capital-framework/builds/327021795?utm_source=github_status&utm_medium=notification

Reply:
Thanks.